tests/fast-integration/monitoring-test/monitoring-test.js

```diff
@@ -0,0 +1,102 @@
+const uuid = require("uuid");
+
+const {
+  assert
+} = require("chai");
+
+const {
+  debug,
+  genRandom,
+  waitForPodWithLabel,
+} = require("../utils");
+
+const {
+  prometheusPortForward,
+  getPrometheusActiveTargets,
+  getPrometheusAlerts,
+  getPrometheusRuleGroups,
+} = require("../monitoring/client")
+
+const {
+  shouldIgnoreTarget,
+  shouldIgnoreAlert,
+  buildScrapePoolSet,
+  assertTimeSeriesExist: assertTimeSeriesExists,
+} = require("../monitoring/helpers");
+
+describe("Monitoring test", function () {
+  const suffix = genRandom(4);
```

Comment:
appName, runtimeName, scenarioName, and runtimeID are not used. Copy & paste?

Reply:
Removed